ui/app/AppLayouts/Chat/panels/GroupChatPanel.qml

```diff
@@ -150,7 +156,7 @@ RowLayout {
          showSortedListOnlyWhenText: true
          toLabelText: qsTr("To: ")
          warningText: qsTr("USER LIMIT REACHED")
-         onTextChanged: sortModel(d.contactsModel)
+         onTextChanged: sortModel(d.buildContactsModel())
```

Comment:
Why do we need here to rebuild again the `contactModel` list (this list should not change)?